Dockerfile

```diff
@@ -20,8 +21,7 @@ COPY Makefile /app/woeip
 COPY Pipfile /app/woeip
 COPY Pipfile.lock /app/woeip
 
-# TODO Switch to environment variable when https://github.com/pypa/pipenv/issues/3278 is resolved.
-RUN pipenv install --dev --system
+RUN make requirements
```

Comment:
`make requirements` calls `pipenv install --dev`. A recent update to `pipenv` lets us use `PIPENV_SYSTEM=1` to tell `pipenv` to use the system Python rather than create a virtualenv. Virtualenvs don't make sense in a Docker container.